.env.example

```diff
@@ -40,3 +40,8 @@ SPEED=200 # TPS = ~2 * SPEED (Default SPEED = 200; TPS = ~400)
 MAX_ACCOUNTS=100000 # number of transactions to be sent
 FUND=true # boolean flag to fund the accounts or not (use true for first time and false for subsequent runs)
 STRESS_DEBUG_LOGS=false # boolean flag to enable debug logs for the stress tests
+
+#EIP-1559 test variables
+BURN_CONTRACT_ADDRESS=0x000000000000000000000000000000000000dead # Burn contract address
+MAX_FEE=30000000009 # Max fee per gas
```

Comment:
Just for me to understand. Where does `30000000009` come from?

Reply:
It's just a value for max fee per gas which was used by default in the[ test script](https://github.com/maticnetwork/eip1559-test-scripts/blob/b7d7246fd371a5d8786cf113b03a19768a4f1eca/testAll.js#L165-L166)